content/components/treeview.mdx

```diff
@@ -0,0 +1,595 @@
+---
+title: Tree view
+---
+
+import {Box} from '@primer/react'
+
+<!-- TODO: remove alt attributes from presentational images before merging -->
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+  A tree view is a hierarchical list of items that may have a parent-child relationship where children can be toggled
+  into view by expanding or collapsing their parent item.
+</Box>
+
+## Anatomy
+
+<img
+  width="960"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/187719332-3ef9aa9e-476f-4e5a-93fa-5fbd203ece4a.png"
+/>
+
+1. **Parent node**: A node that hides and shows child nodes by expanding and collapsing
+2. **End node**: A node with no children
+3. **Nesting level indicator lines**: Each line represents a level of nesting depth
+4. **Chevron**: Toggles a parent node expanded or collapsed and indicates expanded/collapsed state
+5. **Leading visual (optional)**: A visual cue for additional context
+6. **Node label**: The text label for the node
+7. **Trailing visual (optional)**: Same as a leading visual, but at the end
+
+## Content
+
+### Node label
+
+Node labels should be a succinct title for whatever entity the node represents. For example, a file name.
+
+By default, labels are truncated to a single line. However, node labels may be wrapped to multiple lines in cases where the entire label must always be visible.
+
+When a node's text is truncated, it should still be accessible to users via a browser-native tooltip.
```

Comment:
@mperrotti I think one issue can be that most browsers will not display the browser tooltip on focus, if I remember right 😞 Here's a post from Scott O'Hara that I appreciated on this topic: https://www.24a11y.com/2017/the-trials-and-tribulations-of-the-title-attribute/ (although it might be outdated at this point lol)

Reply:
Thank you! The React tooltip also doesn't show when focusing the target, nor does it require the target be focusable 😭

https://primer.style/react/Tooltip

Reply:
Could we wrap the Radix Tooltip component? https://www.radix-ui.com/docs/primitives/components/tooltip

Reply:
@colebemis - we could if other folks are comfortable with that and it's accessible

Reply:
@colebemis, @mperrotti I'm cool with this!